test/distributed/algorithms/quantization/test_quantization.py

```diff
@@ -26,7 +27,7 @@ def _build_tensor(size, value=None, dtype=torch.float, device_id=None):
     if value is None:
         value = size
     if device_id is None:
-        return torch.empty(size, size, size, dtype=dtype).fill_(value)
+        return torch.empty(size, dtype=dtype).fill_(value)
     else:
         return torch.empty(size, size, size, dtype=dtype).fill_(value).cuda(device_id)
```

Comment:
why we don't change the torch.empty here? we should all using `torch.empty(size, dtype=dtype)`?